### PR TITLE
More granular debug5

### PR DESCRIPTION
**Pull request by rebroad**
Builds upon #8484 

Currently the debug.log output is a bit of a mess with few options to configure it to be more readable or to focus on the areas of interest. This pull request helps make this possible by allowing only specific areas of the program debug when selected, and provides a more granular way to increase/decrease verbosity of those areas.
#8729 extends this pull request further

### CODE REVIEW

File: src/init.cpp [hunk 1]
```diff
@@ -423,7 +423,7 @@ std::string HelpMessage(HelpMessageMode mode)
         strUsage += HelpMessageOpt("-limitdescendantsize=<n>", strprintf("Do not accept transactions if any ancestor would have more than <n> kilobytes of in-mempool descendants (default: %u).", DEFAULT_DESCENDANT_SIZE_LIMIT));
         strUsage += HelpMessageOpt("-bip9params=deployment:start:end", "Use given start/end times for specified BIP9 deployment (regtest-only)");
     }
-    string debugCategories = "addrman, alert, bench, coindb, db, http, libevent, lock, mempool, mempoolrej, net, proxy, prune, rand, reindex, rpc, selectcoins, tor, zmq"; // Don't translate these and qt below
+    string debugCategories = "addrman, alert, bench, block, coindb, db, estimatefee, http, libevent, lock, mempool(,rej}, net{,2}, proxy, prune, rand, reindex, rpc, selectcoins, tx, tor, zmq"; // Don't translate these and qt below
```
Comment (paveljanik) [inline src/init.cpp]: Is `mempool(,rej},` correct?

**Comment by paveljanik** [comment]
Why?

File: src/init.cpp (hunk 1 above)
Comment (rebroad) [inline src/init.cpp]: yes, it's a typical notation that expands to mempool, mempoolrej

Comment (paveljanik) [inline src/init.cpp]: `(` and '}'?

Comment (rebroad) [inline src/init.cpp]: ah... my eyesight is failing me!

**Pull request closed without merging**